### PR TITLE
Id 36 center map button

### PR DESCRIPTION
**Pull request by wojtek717**

### CODE REVIEW

**Comment by wojtek717** [comment]
[Centetr map button](https://trello.com/c/UQuwUg07/36-centetr-map-button)

File: kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift [hunk 1]
```diff
@@ -11,23 +11,41 @@ import MapKit
 
 protocol MapViewControllerPresenter: NSObject {
   func centerMap(coordinateRegion: MKCoordinateRegion)
+  func setUpUI()
 }
 
 class MapViewController: UIViewController {
 
+  @IBOutlet weak var centerMapButton: UIButton!
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift]: `private`

File: kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift [hunk 2]
```diff
@@ -11,23 +11,41 @@ import MapKit
...
   override func viewDidLoad() {
     super.viewDidLoad()
 
-   viewModel.centerMapOnUser()
+    viewModel.centerMapOnUser()
+    setUpUI()
+
+  }
+  
+  @IBAction func centerMapButtonPressed(_ sender: UIButton) {
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift]: `private`
Plz also move it higher, under `ibOutlets`

File: kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift [hunk 3]
```diff
@@ -11,23 +11,41 @@ import MapKit
...
   }
 }
 
 extension MapViewController: MapViewControllerPresenter {
   func centerMap(coordinateRegion: MKCoordinateRegion) {
     mapView.setRegion(coordinateRegion, animated: true)
   }
+
+  func setUpUI() {
+    centerMapButtonView.layer.cornerRadius = 32
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewControllers/MapViewController.swift]: When you want to make view circular, it's better to use view height/width to round corners.

**Pull request merged**